app/models/user.rb

```diff
@@ -497,7 +497,7 @@ def profile_image_90
   end
 
   def unsubscribe_from_newsletters
-    return if email.blank?
+    return if email.blank? || (SiteConfig.mailchimp_api_key.blank? && SiteConfig.mailchimp_newsletter_id.blank?)
```

Comment:
Apart from this linme being too long, I think I'd prefer two distinct guard clauses here:

```ruby
return if email.blank?
return if SiteConfig.mailchimp_api_key.blank? && SiteConfig.mailchimp_newsletter_id.blank?
```